src/set_theory/surreal/basic.lean

```diff
@@ -82,6 +82,21 @@ theorem numeric_rec {C : pgame → Prop}
 | ⟨l, r, L, R⟩ ⟨h, hl, hr⟩ :=
   H _ _ _ _ h hl hr (λ i, numeric_rec _ (hl i)) (λ i, numeric_rec _ (hr i))
 
+/-- Relabellings preserve being numeric. -/
+theorem numeric_congr {x y : pgame} (ox : numeric x) (r : relabelling x y) : numeric y :=
+begin
+  induction x using pgame.move_rec_on with x IHl IHr generalizing y,
+  cases r with _ _ L R hL hR,
+  refine (numeric_def y).2 ⟨λ i j, _, λ i, _, λ j, _⟩,
+  any_goals
+  { have H := hL (L.symm i),
+    rw equiv.apply_symm_apply at H },
```

Comment:
Which goals are you actually targetting here? `any_goals` can be fragile, because if a proof stops working on a particular goal then the fixer end up confused about which goals it was supposed to work on.